Error compiling compass on windows

Since version 2.1 i can't compile on windows, on linux it's ok.
```
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T18:29:23+01:00)
Maven home: C:\software\apache-maven\bin\..
Java version: 1.7.0_76, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.7.0_76\jre
Default locale: cs_CZ, platform encoding: Cp1250
OS name: "windows 8.1", version: "6.3", arch: "amd64", family: "windows"
```
```
[ERROR] Compilation of template C:/git/web/src/main/webapp/static/css/sass/NPS-questionary.scss failed: File to import not found or unreadable: compass/css3/opacity.
Load paths:
  C:/Users/user/.m2/repository/nl/geodienstencentrum/maven/sass-maven-plugin/2.4/sass-maven-plugin-2.4.jar!/gems/compass-core-1.0.3/stylesheets
  Compass::SpriteImporter
```

NPS-questionary.scss:
```
@import 'compass/css3/opacity';
```

--- a/pom.xml
+++ b/pom.xml
@@ -130,7 +130,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<compass.version>1.0.3</compass.version>
 		<sass.version>3.4.15</sass.version>
-		<jruby.version>9.0.0.0.pre2</jruby.version>
+		<jruby.version>9.0.0.0.rc1</jruby.version>
 		<maven-libs.version>3.3.3</maven-libs.version>
 		<scss-lint.version>0.40.1</scss-lint.version>
 		<maven.version />

--- a/pom.xml
+++ b/pom.xml
@@ -130,7 +130,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<compass.version>1.0.3</compass.version>
 		<sass.version>3.4.15</sass.version>
-		<jruby.version>1.7.21</jruby.version>
+		<jruby.version>9.0.0.0.pre1</jruby.version>
 		<maven-libs.version>3.3.3</maven-libs.version>
 		<scss-lint.version>0.40.1</scss-lint.version>
 		<maven.version />

--- a/pom.xml
+++ b/pom.xml
@@ -130,7 +130,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<compass.version>1.0.3</compass.version>
 		<sass.version>3.4.15</sass.version>
-		<jruby.version>9.0.0.0.pre1</jruby.version>
+		<jruby.version>9.0.0.0.pre2</jruby.version>
 		<maven-libs.version>3.3.3</maven-libs.version>
 		<scss-lint.version>0.40.1</scss-lint.version>
 		<maven.version />

--- a/pom.xml
+++ b/pom.xml
@@ -130,7 +130,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<compass.version>1.0.3</compass.version>
 		<sass.version>3.4.15</sass.version>
-		<jruby.version>9.0.0.0.rc1</jruby.version>
+		<jruby.version>9.0.0.0.rc2</jruby.version>
 		<maven-libs.version>3.3.3</maven-libs.version>
 		<scss-lint.version>0.40.1</scss-lint.version>
 		<maven.version />

